blossom/templates/app/transcribing.html

```diff
@@ -3,58 +3,252 @@
+            maxZoomPixelRatio: 12,
+        });
+        const simplemde = new SimpleMDE(
+            {
+                autosave: {
+                    enabled: true,
+                    uniqueId: "grafeasTranscriptionField",
+                    delay: 1000,
+                },
+                toolbar: ["bold", "italic", "heading", "|", "quote", "unordered-list", "|", "link", "code", "|", "preview", "side-by-side", "fullscreen", "|", "guide"],
+                element: document.getElementById("transcriptionTextArea"),
+                renderingConfig: {
+                    codeSyntaxHighlighting: true
                 }
-            });
-            var simplemde = new SimpleMDE(
-                {
-                    autosave: {
-                        enabled: true,
-                        uniqueId: "grafeasTranscriptionField",
-                        delay: 1000,
-                    },
-                    toolbar: ["bold", "italic", "heading", "|", "quote", "unordered-list", "|", "link", "code", "|", "preview", "side-by-side", "fullscreen", "|", "guide"],
-                    element: document.getElementById("transcriptionTextArea"),
-                    renderingConfig: {
-                        codeSyntaxHighlighting: true
-                    }
+            }
+        );
+
+        {% if not transcription %}
+            simplemde.value("Put your transcription here!");
+        {% else %}
+            simplemde.value(`{{ transcription }}`)
+        {% endif %}
+
+        fetch('{% url "subredditjsonproxy" %}?s={{ submission.get_subreddit_name }}')
+            .then(response => response.json())
+            .then(data => {
+                modal = document.getElementById('rulesModalBody');
+                let ul = document.createElement('ul');
+                if (Object.keys(data['rules']).length === 0) {
+                    let alert = document.createElement('div');
+                    alert.classList.add(..."alert alert-secondary text-center".split(" "));
+                    alert.setAttribute("role", "alert");
+                    alert.innerText = (
+                        "It looks like {{ submission.get_subreddit_name }} doesn't have any rules published."
+                    );
+                    modal.appendChild(alert);
+
+                } else {
+                    data['rules'].forEach((el) => {
+                        let li = document.createElement('li');
+                        li.classList.add('mt-2');
+                        let strong = document.createElement('strong');
+                        strong.innerText = `${el['violation_reason']}`;
+                        let span = document.createElement('span');
+                        span.innerText = ` - ${el['description']}`;
+                        li.appendChild(strong);
+                        li.appendChild(span);
+                        ul.appendChild(li);
+                    });
+                    modal.appendChild(ul);
```

Comment:
I wonder, if we can use the template syntax here, can we replace this stuff with the templates somehow? Not sure if this is actually possible though